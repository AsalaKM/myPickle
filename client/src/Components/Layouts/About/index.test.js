import React from "react"
import renderer from "react-test-renderer"
import { StaticRouter } from "react-router-dom"

import AboutmyPikle from "./index"

test("AboutmyPikle matches snapshot", () => {
  const context = {}
  const component = renderer.create(
    <StaticRouter context={context}>
      <AboutmyPikle />
    </StaticRouter>
  )
  expect(component).toMatchSnapshot()
})
