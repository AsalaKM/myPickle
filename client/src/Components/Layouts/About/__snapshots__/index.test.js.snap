// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutmyPikle matches snapshot 1`] = `
<div
  className="sc-bdVaJa eFPZsD"
>
  <h2
    className="sc-bwzfXH kCppBb"
  >
    About My Pickle
  </h2>
  <div
    className="sc-bxivhb IZFgF"
  >
    <h4>
      We make finding support easy. As an independent nonprofit community support network, we offer:
    </h4>
    <h4>
      We make finding support easy.
    </h4>
    <h4>
      As an independent nonprofit community support network, we offer:
    </h4>
    <li
      className="sc-htpNat list f5 pl0 KtFsv"
    >
      <p>
        Simplicity – Expertly designed for great user experience
      </p>
      <p>
        Community – Share & find advice & stories
      </p>
      <p>
        Variety – Search public, private, charities & community services
      </p>
    </li>
    <h4>
      And we give back. All our profits are donated to improve access to services.
    </h4>
  </div>
  <a
    className="sc-EHOje cwviPy"
    href="mailto:cat@mypickle.org"
  >
    Click here to contact us and find out more
  </a>
</div>
`;
